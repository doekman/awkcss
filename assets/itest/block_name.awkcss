BEGIN {
	width(30);
}
/^#/ {
	block_name();
}
/^>/ {
	block_name("quote");
	margin(1, 0);
}
