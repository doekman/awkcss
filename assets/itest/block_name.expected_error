‼️ block_name with value '': You need to supply a value
