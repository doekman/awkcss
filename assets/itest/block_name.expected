# This is a title             
                              
> This is a quote.            
> Continuation of the quote.  
> Ending of the quote         
                              
Some more text.               
